spec/javascript_runtime_spec.rb

```diff
@@ -204,4 +204,14 @@
       expect(rt.t(num)).to eq(23)
     end
   end
+
+  context 'with global variables already present' do
+    context = V8::Context.new
```

Comment:
@camertron `context` within a `context`. I was wondering if there was a better way... I suppose the inner `context` occupies the block scope and things work. I suppose there is a way to use the outer `context` in the block. No?